app/dawn-hub/Views/OnboardingView/OnboardingViewContainer.jsx

```diff
@@ -49,9 +49,6 @@ class OnboardingViewContainer extends Component {
 			sendMountActions: false,
 		};
 
-		const { history } = this.props;
-		history.push(`/${ONBOARDING}/${WELCOME}`);
-
```

Comment:
Since we are removing this and users will start at the root by defualt, we need an entry for the root path in the `steps` array in `OnboardingViewContainer#render`